Sources/_NIO1APIShims/NIO1APIShims.swift

```diff
@@ -78,7 +78,57 @@ extension StaticString: Collection {
 extension ChannelPipeline {
     @available(*, deprecated, message: "please use ByteToMessageHandler(myByteToMessageDecoder)")
     public func add<Decoder: ByteToMessageDecoder>(handler decoder: Decoder) -> EventLoopFuture<Void> {
-        return self.add(handler: ByteToMessageHandler(decoder))
+        return self.addHandler(ByteToMessageHandler(decoder))
+    }
+
+    @available(*, deprecated, renamed: "addHandler(_:first:)")
```

Comment:
This doesn't look right: it's pointing to a function that doesn't exist.

Reply:
oops